.env

```diff
@@ -0,0 +1,5 @@
+DB_USERNAME='Library'
+DB_PASSWORD='B@@ks!'
+DB_DATABASE='greatNovels'
+DB_HOST='localhost'
+DB_DIALECT='mysql'
```

Comment:
This file should be git ignored, the whole point of the sample file is so that we don't commit this file with real credentials but a dev knows what variables need to be set.